src/System.Net.ServicePoint/tests/System.Net.ServicePoint.Tests.csproj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
```

Comment:
Actually, we have this in a bunch of projects.  Do they all need to be changed to add a condition?
```
corefx\src>findstr /s /c:"<NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>" *.csproj
System.CodeDom\tests\System.CodeDom.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Collections\tests\Performance\System.Collections.Performance.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Collections.NonGeneric\tests\Performance\System.Collections.NonGeneric.Performance.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.ComponentModel.TypeConverter\tests\Performance\System.ComponentModel.TypeConverter.Performance.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Diagnostics.Debug\tests\System.Diagnostics.Debug.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Diagnostics.TraceSource\tests\System.Diagnostics.TraceSource.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Net.Mail\tests\Functional\System.Net.Mail.Functional.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Net.Mail\tests\Unit\System.Net.Mail.Unit.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Net.Security\tests\UnitTests\System.Net.Security.Unit.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Net.ServicePoint\tests\System.Net.ServicePoint.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Net.WebClient\tests\System.Net.WebClient.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Net.WebProxy\tests\System.Net.WebProxy.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Private.Uri\tests\ExtendedFunctionalTests\System.Private.Uri.ExtendedFunctional.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Runtime.Serialization.Formatters\tests\System.Runtime.Serialization.Formatters.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Runtime.Serialization.Json\tests\Performance\System.Runtime.Serialization.Json.Performance.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Runtime.Serialization.Xml\tests\Performance\System.Runtime.Serialization.Xml.Performance.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Security.Permissions\tests\System.Security.Permissions.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Security.Principal.Windows\tests\System.Security.Principal.Windows.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
System.Transactions\tests\System.Transactions.Tests.csproj:    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
```